src/js/profile/mobile/widget/Calendar.js

```diff
@@ -201,11 +272,30 @@
 			* @member ns.widget.mobile.Calendar
 			* @protected
 			*/
-			prototype._selection = function (element) {
-				if (selectDay.innerHTML != element.innerHTML) {
-					element.classList.add("selection");
-					selectDay.classList.remove("selection");
-					selectDay = element;
+			prototype._selection = function (element, value) {
+				var otherMonthDay,
+					idx;
+
+				if (value != undefined) {
+					otherMonthDay = this.element.querySelectorAll(".currentEnableDay");
+					for (idx = 0; idx < otherMonthDay.length; idx++) {
```

Comment:
I think forEach or 'for of' look better than for loop.

Reply:
I use 'for of'.

done.